README.md

```
@@ -93,6 +93,61 @@ cd ~/ghq/github.com/mikinovation/dotfiles
+# 2. password-store を初期化
+pass init <YOUR_GPG_KEY_ID>
+
+# 3. 既存のストアを private リポジトリから復元する場合
+git clone git@github.com:<you>/password-store.git ~/.password-store
+```
+
+### ディレクトリ規約
+
+環境切替用の変数は `env/<env-name>/<VAR_NAME>` に格納します。
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)